pkg/controller/datavolume/smart-clone-controller.go

```diff
@@ -369,6 +369,17 @@ func newPvcFromSnapshot(name string, snapshot *snapshotv1.VolumeSnapshot, target
 		common.CDILabelKey:       common.CDILabelValue,
 		common.CDIComponentLabel: common.SmartClonerCDILabel,
 	}
+
+	dv, err := getDataVolume(c, client.ObjectKeyFromObject(snapshot))
+	if err != nil {
+		return nil, err
+	}
+	if dv != nil {
+		for k, v := range dv.Labels {
```

Comment:
It seems like we also don't pass the annotations in this case, we probably should